tracer/test/test-applications/integrations/Samples.RuntimeMetrics/Program.cs

```diff
@@ -47,7 +48,19 @@ private static void GenerateEvents()
                 }
 
                 // Sleep for 500ms while creating contention
-                Monitor.TryEnter(SyncRoot, 500);                
+                Monitor.TryEnter(SyncRoot, 500);
+            }
+        }
+
+        private static void GenerateMemoryPressure()
+        {
+            while (true)
+            {
+                // Do some big allocating etc
```

Comment:
Can you add the reason for doing this to the comment? :)
I assume it's to ensure the "committed memory" metric used in the integration test changes over time.